### PR TITLE
feat: add Mistral Instruct model to the catalog

### LINKED ISSUE #1771: Add Mistral Instruct model to the catalog

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #1771

### What does this PR do?

Add a new model to the catalog

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#1771

### How to test this PR?

Test with playground

### CODE REVIEW

File: packages/backend/src/assets/ai.json [hunk 1]
```diff
@@ -303,6 +303,19 @@
       },
       "sha256": "893ae2442b36b2e8e1134ccbf8c0d9bd670648d0964509202ab30c9cbb3d2114",
       "backend": "none"
+    },
+    {
+      "id": "hf.MaziyarPanahi.Mistral-7B-Instruct-v0.3.Q4_K_M",
```
Comment (benoitf) [inline packages/backend/src/assets/ai.json:308]: should it come from https://huggingface.co/mistralai rather than "hf.MaziyarPanahi"

Comment (jeffmaury) [reply packages/backend/src/assets/ai.json:308]: They don't provide GGUF files thus this repo

**Review by benoitf — APPROVED** [review]
I don't know if we have a process to say if we include or not a model like  if we can trust the author or not, if there are no changes from the original model etc but it's a different story.

**Review by gastoner — APPROVED** [review]

**Review by axel7083 — APPROVED** [review]
LGTM

I know we had some idea at some point to include the model converter (.safetensor => gguf) inside ai lab 

- https://github.com/containers/podman-desktop-extension-ai-lab/issues/542
- https://github.com/containers/ai-lab-recipes/tree/main/convert_models

Maybe we should re-evaluate this ?

**Comment by jeffmaury** [comment]
> LGTM
> 
> I know we had some idea at some point to include the model converter (.safetensor => gguf) inside ai lab
> 
>     * [Integrate Model converter for running models without GGUF file #542](https://github.com/containers/podman-desktop-extension-ai-lab/issues/542)
> 
>     * https://github.com/containers/ai-lab-recipes/tree/main/convert_models
> 
> 
> Maybe we should re-evaluate this ?

That needs to be tested first as GGUF contains model metadata (template,...) that are not present in the safe tensor format

**Pull request merged**